docs/ops/deployment/kubernetes.md

```diff
@@ -264,6 +264,24 @@ spec:
     component: jobmanager
 {% endhighlight %}
 
+`taskmanager-query-state-service.yaml`. Optional service, that exposes the taskmanager `query-state` port as public Kubernetes node's port.
```

Comment:
should we mention this service in `## Deploy Flink cluster on Kubernetes` chapter?
like we describe `jobmanager-rest-service.yaml`:
- after description of access REST API, we could mention that the `query-state` port can be accessed in a similar way.
- add the optional `kubectl create/delete -f taskmanager-query-state-service.yaml` command